var/spack/repos/builtin/packages/valgrind/package.py

```diff
@@ -73,12 +75,13 @@ class Valgrind(AutotoolsPackage, SourcewarePackage):
 
     def configure_args(self):
         spec = self.spec
-        options = []
+        options = [self.enable_or_disable('libs')]
```

Comment:
enable_or_disable returns an array itself
```suggestion
        options = self.enable_or_disable('libs')
```